src/stores/PersonStore.ts

```diff
@@ -31,6 +41,35 @@ export type Person = {
   personalnummer?: string | null;
 };
 
+type PersonenWithRolleAndZuordnung = {
+  rollen: string;
+  administrationsebenen: string;
+  klassen: string;
+  person: Person;
+}[];
+
+export type Uebersicht =
```

Comment:
Außerhalb dieses Files ist nicht klar, was für eine Übersich gemeint ist.

```suggestion
export type PersonenUebersicht =
```